src/ychaos/testplan/verification/__init__.py

```diff
@@ -145,6 +144,56 @@ class NoOpConfig(SchemaModel):
     pass
 
 
+class OpenTSDBVerification(SchemaModel):
+    """
+    The OpenTSDB Verification Plugin gets the metrics from an OpenTSDB server and compares it with the
+    provided comparison parameters in the testplan. If the condition passes
+    """
+
+    url: AnyHttpUrl = Field(
+        ..., description="The OpenTSDB server URL to get the metrics from"
+    )
+
+    method: str = Field(
+        default="GET",
+        description="The HTTP method used to query the metrics from OpenTSDB server.",
+        examples=["GET", "POST"],
+    )
+
+    query: Dict[str, Any] = Field(
+        default=dict(), description="The OpenTSDB query sent to the server."
+    )
+
+    criteria: List[MultipleConditionalsMetricsVerificationCriteria] = Field(
+        default=list(),
+        description=(
+            "Metrics verification criteria without state information."
+            "All the criteria part of this list must pass for the verification to be successful"
+        ),
+    )
+
+    state_bound_criteria: List[StateBoundMetricsVerificationCriteria] = Field(
+        default=list(),
+        description=(
+            "Metrics verification criteria with state. "
```

Comment:
nit:
keeping it consistent with the rest
```suggestion
            "Metrics verification criteria with state."
```